src/uinput.rs

```diff
@@ -15,6 +15,10 @@ pub struct UInputDevice {
 unsafe impl Send for UInputDevice {}
 
 impl UInputDevice {
+    fn raw(&self) -> *mut raw::libevdev_uinput {
```

Comment:
I don't understand why we need this method. Isn't it redundant with `<UinputDevice as LibevdevWrapper>::raw()`?

Reply:
The same question applies to most methods in this file

Reply:
Ignore this comment - I thought LibevdevWrapper was supposed to abstract over `UInputDevice`s in addition to `UninitDevice`s and `Device`s